File: 1. Model/Evento.py


class Evento:

    def __init__(self, data, horaInicio, horaFim, brincadeiras, localEvento, economia):
        self.data         = data
        self.horaInicio   = horaInicio
        self.horaFim      = horaFim
        self.brincadeiras = brincadeiras
        self.localEvento  = localEvento
        self.economia     = economia



    



    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getData(self):
        return self.data

    def getHoraInicio(self):
        return self.horaInicio

    def getHoraFim(self):
        return self.horaFim

    def getBrincadeiras(self):
        return self.brincadeiras

    def getLocalEvento(self):
        return self.localEvento

    def getEconomia(self):
        return self.economia


    ##SETS
    def setData(self, d):
        self.data = d

    def setHoraInicio(self, hi):
        self.horaInicio = hi

    def setHoraFim(self, hf):
        self.horaFim = hf

    def setBrincadeiras(self, b):
        self.brincadeiras = b

    def setLocalEvento(self, le):
        self.localEvento = le

    def setEconomia(self, e):
        self.economia = e
    
    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Evento.java
package model;

//import java.util.Date;



public class Evento {

	private String economia;

	private String data;

	private String horaInicio;
	private String horaFim;

	private String brincadeiras;
	private String localDeEvento;



	public Evento(String data, String horaInicio, String horaFim, String brincadeiras, String localDeEvento, String economia) {
		this.data 	  	   = data;
		this.horaInicio    = horaInicio;
		this.horaFim 	   = horaFim;
		this.brincadeiras  = brincadeiras;
		this.localDeEvento = localDeEvento;
		this.economia 	   = economia;
	}



	// Gets
	public String getData() {
		return this.data;
	}

	public String getHoraInicio() {
		return this.horaInicio;
	}

	public String getHoraFim() {
		return this.horaFim;
	}

	public String getBrincadeiras() {
		return this.brincadeiras;
	}

	public String getLocalDeEvento() {
		return this.localDeEvento;
	}

	public String getEconomia() {
		return this.economia;
	}



	// Sets
	public void setData(String da) {
		this.data = da;
	}

	public void setHoraInicio(String hi) {
		this.horaInicio = hi;
	}

	public void setHoraFim(String hf) {
		this.horaFim = hf;
	}

	public void setBrincadeiras(String br) {
		this.brincadeiras = br;
	}

	public void setLocalEvento(String lde) {
		this.localDeEvento = lde;
	}

	public void setEconomia(String ec) {
		this.economia = ec;
	}
}
